### DISCUSSION TITLE
Sensor setup problem

### DISCUSSION THREAD

**Discussion by Kenny-CN** [zephyrproject-rtos/zephyr#40649] [Q&A] [answered]
I am using LIS2DH sensor, and want to add shutdown function. But the lis2dh driver in zephyr does not seem to support turning off the sensor. 

in "zephyr\drivers\sensor\lis2dh\lis2dh.c" file:
![image](https://user-images.githubusercontent.com/79444755/143346569-8ab81222-4999-4fad-9abf-de43afc2168a.png)

Turn off the sensor by setting ODR to 0. But the driver does not seem to allow setting odr to 0.
So how do I turn off the sensor?

**Comment by erwango** [comment]
@avisconti ^^

**Comment by avisconti** [comment, marked as answer]
See #35486
